### ISSUE TITLE
Dapper.Contrib + Async + Cancellation

### ISSUE THREAD

**Issue by chriskolenko** [StackExchange/dapper-dot-net#479]
Any plans to support Cancellation with Insert, Update, Delete etc, methods?

**Comment by alexsandro-xpt** [comment]
Just like a rollback transation for long duration query?

**Comment by chriskolenko** [comment]
My apologies, it should have said CancellationToken. I believe CommandDefinition has support for a Cancellation Token.

**Comment by NickCraver** [comment]
Yes, we plan to add these optional parameters, but it'll have to be in the next major release as it is a breaking API change. I expect we'll do such a release with .Net Core releases, possibly in alpha/beta states earlier so you can grab them off NuGet well before .Net Core RTM lands.

I'll tag this issue and leave it open for tracking.
